teacher.rb

```diff
@@ -0,0 +1,11 @@
+require_relative 'Person'
+class Teacher < Person
+  def initialize(specialization, age, name = 'unknown', parent_permission: false)
+    super(name, age, parent_permission)
+    @specialization = specialization
+  end
```

Comment:
- [ ] The parent constructor accepts `(age, name, parent_permission)` as parameter and you are passing `(name, age, parent_permission)`, It will swap `age` with the `name`. Please fix this issue.

Reply:
@thecodechaser!
Thank you for reviewing my code. I will change it.